Gems/Atom/Asset/Shader/Code/Source/Editor/ShaderBuilderUtility.cpp

```diff
@@ -885,6 +884,36 @@ namespace AZ
                 return AZ::Success(AZStd::move(listOfFilePaths));
             }
 
+            AZStd::string GetPlatformNameFromPlatformIdentifier(const AZStd::string& platformIdentifier)
```

Comment:
Recommend PAL-ifying this as there will be platforms supported not hosted in the main repo

Reply:
Palified.